js/5etools-spells.js

```diff
@@ -30,7 +30,7 @@ function d20plusSpells () {
 		if (url && url.trim()) {
 			const handoutBuilder = playerMode ? d20plus.spells.playerImportBuilder : d20plus.spells.handoutBuilder;
 
-			DataUtil.loadJSON(url).then(async (data) => {
+			DataUtil.spell.loadJSON(url).then(async (data) => {
```

Comment:
`DataUtil.spell.loadJSON` has no arguments; it just loads all the site's spells. This is therefore broken for homebrew, and nonsensical for site URLs besides

We should do something similar to the changes made for classes (where we remove the "select individual URL for site source" part and just allow the user to select "Official" or "[homebrew URL]"